src/pages/styles/index.css

```diff
@@ -24,9 +24,9 @@ main {
     margin-top: 0 !important;
 }
 
-.ant-layout {
+/* .ant-layout {
     background: none !important;
-}
+} */
```

Comment:
best to `rm -rf` code instead of commenting it out

Reply:
Yeah I was waiting for this - deleted all the comments I could find, but I guess I missed this one.  Thank you!